src/main/java/com/questhelper/helpers/quests/zogreflesheaters/ZogreFleshEaters.java

```diff
@@ -299,7 +301,8 @@ public void setupSteps()
 		bringSignedPortraitToZavistic.addDialogStep("I'm here about the sicks...err Zogres");
 		bringSignedPortraitToZavistic.addDialogStep("I have some items that I'd like you to look at.");
 
-		goUpToSithAgain = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0), "Go upstairs to Sithik upstairs in north Yanille.", strangePotion);
+		goUpToSithAgain = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0),
+			"Go upstairs to Sithik in a house north or the Wizards' Guild.", strangePotion);
```

Comment:
Applying the same suggestion as above
```suggestion
			"Go upstairs and speak to Sithik in the house north of the Wizards' Guild.");
```